server/schema/types/root_query_type.js

```diff
@@ -22,12 +22,14 @@ const CoffeeShop = mongoose.model("coffeeShops")
 const selectorInput = new  GraphQLInputObjectType({
   name: 'Selectors',
   fields: {
-    type: { type: GraphQLString },
```

Comment:
Why did you remove the type selector? Did we take that off the attributes for coffee
shop?

Reply:
Although I'm not sure if this is even being used any more so this may be a non-issue

Reply:
I changed it back (update in the new branch -frontend favorite, but as John set up the search feature, we actually don't use  that in the backend anymore.

Reply:
Please just ignore this pull request as they will be updated in the new branch 'frontend-favorite'